src/main/routes/routes.ts

```diff
@@ -141,6 +141,10 @@ export default function(app: Application): void {
   app.post('/subscription-urn-search', ensureAuthenticated, app.locals.container.cradle.subscriptionUrnSearchController.post);
   app.get('/subscription-urn-search-results', ensureAuthenticated, app.locals.container.cradle.subscriptionUrnSearchResultController.get);
   app.post('/unsubscribe-confirmation', ensureAuthenticated, app.locals.container.cradle.unsubscribeConfirmationController.post);
+  app.get('/media-account-review', ensureAuthenticated, app.locals.container.cradle.mediaAccountReviewController.get);
+  app.get('/media-account-review/image', ensureAuthenticated, app.locals.container.cradle.mediaAccountReviewController.getImage);
+  app.post('/media-account-review/approve', ensureAuthenticated, app.locals.container.cradle.mediaAccountReviewController.approve);
+  app.post('/media-account-review/reject', ensureAuthenticated, app.locals.container.cradle.mediaAccountReviewController.reject);
```

Comment:
these are restricted admin paths 👇🏼 👇🏼 👇🏼

Reply:
Moved these down into the restricted paths section